elements/bulbs-dfp/bulbs-dfp.js

```diff
@@ -38,13 +36,25 @@ export default class BulbsDfp extends BulbsHTMLElement {
       },
     });
 
-    let intervalLength = parseFloat(this.getAttribute('refresh-interval', 10));
-    this.refreshInterval = window.setInterval(this.handleInterval, intervalLength);
+    this.adUnitData = this.adsManager.units[this.dataset.adUnit];
+
+    if (!this.adUnitData.refreshDisabled &&
+        (this.hasAttribute('refresh-interval') ||
```

Comment:
@collin @spra85 do we still need this to be configurable as an attribute? Otherwise I'll remove the attribute-specific logic here and pull the value only from ads manager unit data.